docs/data-ai/ai/run-inference.md

```diff
@@ -103,10 +110,47 @@ output_tensors, err := myMLModel.Infer(context.Background(), input_tensors)
 {{% /tab %}}
 {{< /tabs >}}
 
-After adding a vision service, you can use a vision service API method with a classifier or a detector to get inferences programmatically.
-For more information, see the ML Model and Vision APIs:
+{{% /tab %}}
+{{< /tabs >}}
+
+## Cloud inference
+
+Cloud inference enables you to run machine learning models in the Viam cloud, instead of on a local machine.
+Cloud inference runs on powerful servers with more computing power than most edge devices.
+Because cloud inference has access to more computing power, you can benefit from:
```

Comment:
```suggestion
Cloud inference enables you to run machine learning models in the Viam cloud, instead of on a local machine.
Cloud inference often provides more computing power than edge devices, so you can benefit from:
```